extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -25,6 +25,32 @@ define('wikia.articleVideo.featuredVideo.tracking', ['wikia.tracker'], function
 		window.guaSetCustomDimension(36, currentVideo.tags);
 	}
 
+	/**
+	 * Comscore Video Metrix tracking
+	 * C1 (identifier of content) = 1
```

Comment:
what does C* mean? where is it used?

Reply:
These are comscore tracking params, you can think of these as of GA params, but predefined by the tracking software vendor. Documentation: https://wikia-inc.atlassian.net/secure/attachment/116280/comScoreImplementationSpecs-Video.pdf